src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  width: 96px;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &--4 :nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
Ensure that the modifier class `.stars--4` is written to be easily extendable for different amounts of active stars. It should be possible to create `.stars--1`, `.stars--2`, etc., without adding additional CSS for each one. Consider using `@for` directive or another method to generate the necessary CSS.

Reply:
Not fixed